### PR TITLE
Rescheduling email redux

### PR DESCRIPTION
**Pull request by jiannis**

### CODE REVIEW

File: confirmation.html [hunk 1]
```diff
@@ -189,11 +189,11 @@
...
     <tr>
         <td>
-            <h1>Appointment booking confirmation</h1>
+            <h1>You've successfully booked an appointment</h1>
 
-            <p>Hi Liz</p>
+            <p>Dear Liz</p>
 
-            <p>Thank you for booking a telephone guidance appointment with Pension Wise.
+            <p>thank you for booking a telephone guidance appointment with Pension Wise.
```
Comment (andrewgarner) [inline confirmation.html]: Why a lowercase `t`?

File: confirmation.txt [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
-Appointment booking confirmation
---------------------------------
+Appointment confirmation
+------------------------
 
-Hi Liz
+Dear Liz
 
-Thank you for booking a telephone guidance appointment with Pension Wise. Your appointment is confirmed for:
+thank you for booking a telephone guidance appointment with Pension Wise. Your appointment is confirmed for:
```
Comment (andrewgarner) [inline confirmation.txt]: Ditto

File: confirmation.html (hunk 1 above)
Comment (jiannis) [inline confirmation.html]: There's [the way](http://english.stackexchange.com/a/195332) of the queen, the president and the pope and then there's @hvonhaaren

Comment (alexkouz) [inline confirmation.html]: Queen's English please gentlemen. Comma and uppercase T.

Comment (jiannis) [inline confirmation.html]: @alexkouz Yes ma'am
@hvonhaaren Mind your Cs and Ts

**Pull request merged**